vfs/vfscache/item.go

```diff
@@ -1197,7 +1226,16 @@ func (item *Item) readAt(b []byte, off int64) (n int, err error) {
 }
 
 // WriteAt bytes to the file at off
+// Protected against Reset using preAccess/postAccess
 func (item *Item) WriteAt(b []byte, off int64) (n int, err error) {
+	item.preAccess()
```

Comment:
No need to split the function here either

Reply:
Fixed